andela/mont-blanc-rc
# #152798777 Digital Products

#### What does this PR do?
Implements digital product addition.

#### Description of Task to be completed?
It makes it possible to add digital products and display digital products orders efficiently.

#### How should this be manually tested?
+ An admin should create products and assign them to their respective type(digital or nondigital).
+ A user needs to complete an order successfully.
+ Users should check their order.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
`#152798777`

#### Screenshots 
+ Add Product
![image](https://user-images.githubusercontent.com/23665997/33657412-89b012a8-da79-11e7-9524-5f53b8ac24b2.png)

+ View Order
![image](https://user-images.githubusercontent.com/23665997/33657327-3f06ca94-da79-11e7-9c37-089a333a7b34.png)

#### Questions:
None
